src/tests/sysdb-tests.c

```diff
@@ -1095,6 +1105,9 @@ START_TEST(test_user_group_by_name)
      * ldap provider differently with auto_private_groups.
      */
     test_ctx->domain->provider = discard_const_p(char, "ldap");
+    if (!local_provider_is_built()) {
```

Comment:
On the other hand, this test might be handy. Since some functions differ in functionality with mpg enabled domain, I think we want to have a test. But I think just adding a unit test with mpg=true is simpler and more error proof than having an if-else branch based on how sssd is built.